english/explore-outside/journey-to-flaming-tower/jellybus/ask-person-about-destination/ask-person-about-mission/getonbus/journey-to-flaming-tower/journey-to-flaming-tower.md

```diff
@@ -0,0 +1 @@
+The bus sets off for the flaming tower with you and your friend. Rounding a sudden bend the road ran straight into a large donut through which the jelly bus could not fit. You may have to eat a hole through the donut.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.